specification/batch/data-plane/Microsoft.Batch/2017-09-01.6.0/BatchService.json

```diff
@@ -12272,6 +12334,11 @@
           "title": "The exit code of the program specified on the task command line.",
           "description": "This parameter is returned only if the task is in the completed state. The exit code for a process reflects the specific convention implemented by the application developer for that process. If you use the exit code value to make decisions in your code, be sure that you know the exit code convention used by the application process. Note that the exit code may also be generated by the compute node operating system, such as when a process is forcibly terminated."
         },
+        "containerInfo": {
+          "$ref": "#/definitions/TaskContainerExecutionInformation",
+          "title": "Description of the container under which the task runs.",
```

Comment:
Is it really a description?  Perhaps just "Information about..."?

Reply:
Fixed across all descriptions